app/controllers/cases_controller.rb

```diff
@@ -285,17 +285,14 @@ def confirm_respond
 
   def search
     @query = params[:query]
-    @current_tab_name = 'search'
     if @query.present?
       @query.strip!
       @cases = policy_scope(Case::Base).search(@query).page(params[:page]).decorate
       if @cases.empty?
-        flash.now[:alert] = 'No cases found'
+        @cases = nil
       end
```

Comment:
I will change this when I merge in my changes, but it's pointless to say 
``` if @casess.empty?
    @cases = nil
```

and then in the view write 

``` if !@cases.nil? ```

better to leave it empty in the controller, and test ```if @cases.any?``` in the view.  Much clearer what is actually going on that way.